<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="graphGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>MatplotLib</string>
    </property>
    <widget class="QRadioButton" name="radioPlot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioPie">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pie</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioBar">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>390</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bar</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>29</y>
       <width>561</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="matplot_vertical"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="dataGroup">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>371</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Group</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>351</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editX"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editY"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addDataBtn">
        <property name="text">
         <string>Add Data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
